### PR TITLE
Update events handbook for MKCC after NA 2023 event

### PR DESCRIPTION
**Pull request by nimbinatus**
Updating the Contributor Summit events handbook with renaming of the SIG Meet & Greet to Meet the Kubernetes Contributor Community (MKCC) and added notes and updates from this past run at KubeCon/CloudNativeCon NA 2023.

/assign @nimbinatus
/sig contributor-experience
/area contributor-summit

### CODE REVIEW

File: events/events-team/content/meet-and-greet.md [hunk 1]
```diff
@@ -20,116 +20,125 @@ Aside from the SIG leads, this event is run by a Meet & Greet Coordinator (MGC),
...
 
 ## Preparation & Timeline
 
 ### 2-3 Months Before: Room request and SIG gear
 
-Request a large room for the M&G from the CNCF Contributor Summit Liaison.  This will be a large room, set with at least 10 round tables.
+Request a large room for MKCC from the CNCF Contributor Summit Liaison. This will be a large room, set with at least 10 round tables.
 
-The gear for the M&G consists of table sign stands for each SIG, a bag of buttons for each SIG, and Kubernetes Contributor badge ribbons for folks who might not have gotten one.  Some of this gets used/goes missing each KubeCon, so ask CNCF staff to check that:
+The gear for MKCC consists of table sign stands for each SIG, a bag of buttons for each SIG, and Kubernetes Contributor badge ribbons for folks who might not have gotten one. Some of this gets used/goes missing each KubeCon. As the MC, you will get access to a spreadsheet to fill out to identify which SIGs are active and how many buttons should be present for each SIG. Use this sheet to ask CNCF staff to check the geat. Ensure, though the sheet, that:
```
Comment (stmcginnis) [inline events/events-team/content/meet-and-greet.md:41]: s/geat/gear/

File: events/events-team/content/meet-and-greet.md [hunk 2]
```diff
@@ -20,116 +20,125 @@ Aside from the SIG leads, this event is run by a Meet & Greet Coordinator (MGC),
...
 
-Some teams that are not SIGs or WGs may also participate, e.g. ClusterAPI or the CoCC.  There is no specific policy on this, deal with it on a case-by-case basis. Generally speaking, we want to say yes to teams that actually plan to show up as a group. In the rest of this document, "SIGs" should be considered to refer to "SIGs, WGs, and Teams".
+Some teams that are not SIGs or WGs may also participate, e.g. ClusterAPI or the CoCC. There is no specific policy on this, deal with it on a case-by-case basis. Generally speaking, we want to say yes to teams that actually plan to show up as a group. In the rest of this document, "SIGs" should be considered to refer to "SIGs, WGs, and Teams".
 
 ## Time and Location
 
-The M&G is generally held during lunch period on one of the main Kubecon days.  CNCF staff decides the exact schedule in order to not conflict with other lunchtime events.
+MKCC is generally held during lunch period on one of the main Kubecon days. CNCF staff decides the exact schedule in order to not conflict with other lunchtime events.
```
Comment (stmcginnis) [inline events/events-team/content/meet-and-greet.md:33]: If you do an update, might be good to proper-case KubeCon like above. But if not, meh.

**Review by stmcginnis — COMMENTED** [review]
/lgtm

A couple minor things pointed out, but nothing that's too concerning.

Go team single space!

File: events/events-team/content/meet-and-greet.md (hunk 2 above)
Comment (nimbinatus) [reply events/events-team/content/meet-and-greet.md]: Agreed, and fixed :) Thanks!

**Comment by nimbinatus** [comment]
Ready for approval :)

/assign @mrbobbytables

**Review by mrbobbytables — COMMENTED** [review]
thanks!
/label tide/merge-method-squash
/approve

**Pull request merged**